bundle/config/root.go

```diff
@@ -321,6 +321,11 @@ func (r *Root) MergeTargetOverrides(name string) error {
 		return err
 	}
 
+	root, err = normalizeVariableLookupOverrides(root, target)
```

Comment:
Maybe a better name would be `unsetDefaultForVariableLookupOverrides`? `normalize` does not seem like the appropriate verb here.